frontend/src/Auth.purs

```diff
@@ -8,9 +8,10 @@ import Data.Auth.Token as Token
 import Data.Either (hush)
 import Data.Maybe (Maybe(..))
 import Data.Route (Route(..), goTo)
-import Effect.Class (class MonadEffect)
-import LocalStorage (HasStorage)
+import Effect.Class (class MonadEffect, liftEffect)
+import LocalStorage (HasStorage, getStorage)
 import LocalStorage as Storage
+import Web.Storage.Storage (setItem)
```

Comment:
The idea was that `Auth` knows nothing about `Web.Storage.Storage` but instead only uses `LocalStorage`s API (functions): `readKey`, `writeKey`.

Reply:
okay, gonna rework this one

Reply:
done!